packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -2982,6 +3032,7 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
       visitor(foregroundChild);
     if (backgroundChild != null)
       visitor(backgroundChild);
+    super.visitChildren(visitor);
   }
```

Comment:
`RenderEditable` also overrides `redepthChildren`. Should that method be updated as well?